tests/e2e/login/privacy-policy.spec.js

```diff
@@ -126,7 +127,13 @@ describe('Privacy policy', () => {
   });
 
   describe('for a french offline user', () => {
+    let passed = false;
     afterEach(async () => {
+      if (!passed) {
+        // I suspect this test is failing because of a conflict.
```

Comment:
Added issue: https://github.com/medic/cht-core/issues/6960

Reply:
I keep running into things that protractor/jasmine do not does out of the box that other frameworks I've used do. Like afterEach having deails about the spec that ran. IE pass/fail. Nothing to change just a rant.

Reply:
yea, 100% agree. 
The framework does use this info though. If a beforeEach or afterEach fails, they appear as fails attached to the test they ran for.